docker/brew.Dockerfile

```diff
@@ -162,6 +162,9 @@ RUN tar xzf $TARBALL_PATH/skeleton.tar.gz; tar xzf $TARBALL_PATH/bundle.tar.gz;
 # COPY $EXTERNAL_SOURCE_NESTED/app-config*.yaml ./
 # COPY $EXTERNAL_SOURCE_NESTED/dynamic-plugins.default.yaml ./
 
+# Downstream only - fix for https://issues.redhat.com/browse/RHIDP-728
+WORKDIR /opt/app-root/src/
+
 # Install production dependencies
 # hadolint ignore=DL3059
 RUN $YARN install --frozen-lockfile --production --network-timeout 600000
```

Comment:
you are right; this could have some unintended consequences that I did not realized :-(

I guess at this point it might be safer to go with the `mkdir` approach as a hotfix for 1.0.x and than look for a better longterm solution later